submissions/emmanuelEsuong/css/styles-mediaQuery.css

```diff
@@ -0,0 +1,264 @@
+/* media-query (tablet)*/
```

Comment:
I don't think you need split your css files. You should merge them together.

Reply:
It's more easier for me to understand using separate files.

Reply:
@ElJentle You would have to style more than one page going forward and you can't be creating separate files for them. Add the media query at the end you css file that way, you don't mix them up and you can add comments for where the media query starts and ends.

Reply:
Updated all the corrections. 

The bootstrap link on the page is for the social media icons i.e facebook, twitter, YouTube and email.  
Please, how can I get this icons links on the page since I'm to delete the bootstrap.